lib/msal-core/src/ScopeSet.ts

```diff
@@ -54,7 +56,8 @@ export class ScopeSet {
      */
     // TODO: Rename this, too generic name for a function that only deals with scopes
     static removeElement(scopes: Array<string>, scope: string): Array<string> {
-        return scopes.filter(value => value !== scope);
+        const scopeVal = scope.trim().toLowerCase();
```

Comment:
is there a way to have this be encapsulated by trimAndConvertToLowerCase so we are not duplicating this logic?

Reply:
done.